upgrade/helpers/tasks.py

```diff
@@ -316,6 +317,26 @@ def post_upgrade_test_tasks(sat_host):
     execute(lambda: run('sed -i -e \'/:level: / s/: .*/: '
                         'debug/\' /etc/foreman/settings.yaml'), host=sat_host)
     execute(lambda: run('katello-service restart'), host=sat_host)
+    # Execute capsule sync task , after the upgrade is completed
+    if cap_hosts:
+        execute(capsule_sync, cap_hosts, host=sat_host)
+
+
+def capsule_sync(cap_hosts):
```

Comment:
This function will be used to sync user  specified and internal capsule , which is looping in through capsule hosts.

We already had a capsule sync function, for template creation task, I just re-fact it , to be used by general automation